airfoil/lib/airfoil.rb

```diff
@@ -12,16 +12,16 @@
 
 module Airfoil
   class << self
-    def create_stack
+    def create_stack(logger)
       # ensure that STDOUT streams are synchronous so we don't lose logs
       $stdout.sync = true
 
       Signal.trap("TERM") do
         # We can't use the Rails logger here as the logger is not available in the trap context
-        puts "Received SIGTERM, shutting down gracefully..." # rubocop:disable Rails/Output
```

Comment:
I guess this is fine, though _in a Rails context_, rubocop is gonna flag this.

Reply:
![image](https://github.com/highwingio/airfoil/assets/34034891/b9767abe-3c98-4c66-85a7-41f86344be8c)